open_humans/views.py

```diff
@@ -372,28 +334,31 @@ def get_context_data(self, **kwargs):
         project_permissions = None
         granted_permissions = None
         permissions_changed = False
+
         if 'project_id' in self.activity:
             project = DataRequestProject.objects.get(
                 pk=self.activity['project_id'])
 
             project_permissions = {
                 'share_username': project.request_username_access,
-                'share_sources': project.request_sources_access,
                 'all_sources': project.all_sources_access,
                 'returned_data_description': project.returned_data_description,
             }
             if self.activity['is_connected']:
                 project_member = project.active_user(self.request.user)
+                granted_sources = project_member.granted_sources.all()
                 granted_permissions = {
                     'share_username': project_member.username_shared,
-                    'share_sources': project_member.sources_shared,
```

Comment:
removing this is breaking part of the rendered template (it doesn't list sources that were authorized)